dsl_parser/functions.py

```diff
@@ -49,6 +49,13 @@ def _register_entry_point_functions():
         register(fn=entry_point.load(), name=entry_point.name)
 
 
+def _is_function(value):
```

Comment:
In the `parse` function there's a similar check, perhaps use `_is_function` there as well..?